app/controllers/ToDoController.php

```diff
@@ -42,6 +42,18 @@ public function createTaskAction(){
         
         } 
     }
+    
+    // READ ALL TASKS
+	public function showAllTasksAction(){
+        if (($_SERVER['REQUEST_METHOD'] == 'POST')) {
```

Comment:
in this case, I would advise you to use the strict comparison comparator, so that you have the same value and type.